falcon/constants/media_types.py

```diff
@@ -0,0 +1,10 @@
+MEDIA_JSON = 'application/json; charset=UTF-8'
+MEDIA_HTML = 'text/html; charset=utf-8'
+MEDIA_JS = 'text/javascript; charset=utf-8'
+MEDIA_XML = 'application/xml'
+MEDIA_TEXT = 'text/plain; charset=utf-8'
+MEDIA_JPEG = 'image/jpeg'
+MEDIA_PNG = 'image/png'
+MEDIA_YAML = 'application/x-yaml'
```

Comment:
Looks like use of x- is strongly discouraged by RFC : https://tools.ietf.org/html/rfc6838

I will use later instead. If users need to use "application/x-msgpack" they can always set it in media_type.